### PR TITLE
Hacktoberfest cleanup

### PR DESCRIPTION
**Pull request by KillianSmith0**
Just a small code cleanup. 
Making things a bit swiftier. 
Tried to improve performance of the viewController, by removing the `tableView.reloadData()` calls
Made variables immutable where possible

### CODE REVIEW

File: CountryPicker/CountryPicker/CountryManager.swift [hunk 1]
```diff
@@ -31,35 +33,27 @@ open class CountryManager {
...
-            throw "Countries could not be loaded from plist , please check path"
+            throw "Missing array of countries plist in CountryPicker"
         }
     }
 
     func allCountries() -> [Country] {
         return countries
     }
-
-    class func country(with code: String) -> Country {
```
Comment (SURYAKANTSHARMA) [inline CountryPicker/CountryPicker/CountryManager.swift]: Please tell why you consider this function to be deleted.

Comment (KillianSmith0) [reply CountryPicker/CountryPicker/CountryManager.swift]: I couldn't find any uses of it across the project?

Comment (SURYAKANTSHARMA) [reply CountryPicker/CountryPicker/CountryManager.swift]: Cool. Great 💯

Comment (KillianSmith0) [reply CountryPicker/CountryPicker/CountryManager.swift]: 🔥👌🏼

**Pull request merged**